skynet/modules/ttt/openai_api/app.py

```diff
@@ -20,6 +21,7 @@
 
 def initialize():
     if not use_vllm:
+        app.include_router(slim_router)
```

Comment:
do we also want to do it if there is a local llama? I'd only add it if `use_oci` is true